### DISCUSSION TITLE
how to use the tensorflow object detection with sahi?

### DISCUSSION THREAD

**Discussion by dsnsabari** [obss/sahi#911] [General]
I trained the SSD mobilenet v2 object detection model and I want to use the sahi to improve the mAP. can you tell me how to do it?

**Comment by MatheusLealAquino** [comment]
Hey @dsnsabari, do you have any success?

**Reply by kadirnar** [reply]
@dsnsabari , @MatheusLealAquino 

https://github.com/kadirnar/Yolov7-SAHI/blob/main/demo/inference_for_tensorflowhub.ipynb

**Comment by fcakyon** [comment]
@dsnsabari @MatheusLealAquino Unfortunately, currently there is no official support for tensorflow models in sahi
